.travis.yml

```diff
@@ -44,6 +44,12 @@ jobs:
       script:
         - $FUNCTIONAL_TEST bootstrap run bugs
 
+    - name: "regression test for bootstrap bugs (second set)"
```

Comment:
Why have you added a new test execution? Why not just use the first `regression test for bootstrap bugs`?

Reply:
To deploy unique ssh key on each node.
Since from behave(running from one node), it's hard to generate unique ssh key on each node without disturb ssh connection